prime-router/docs/proposals/0009-internal-tools-migration-to-superset.md

```diff
@@ -0,0 +1,50 @@
+## Background
+
+The PRIME ReportStream uses Metabase for business analytics. Metabase is a business intelligence tool that allows for the exploration, investigation, and visualization of data. With this internal tool, our team is able to generate charts and dashboards from our stored data. Even though this instrument provides quality charts, it also has its shortcomings:
+
+### Limitations 
+
+- Query complexity 
+    - Limited to simple queries only
```

Comment:
?  I haven't tried, but I don't have any sense there's a limitation on query complexity in Metabase

Here, as a counter example, is a query currently on our detailed tracker dashboard in Metabase:
```
select PARENT.report_id, CHILD.item_count, PARENT.receiving_org, PARENT.receiving_org_svc,
       date_trunc('second', PARENT.created_at) CREATED,
       date_trunc('second', CHILD.created_at) SENT, CHILD.downloaded_by, PARENT.body_format
from report_file PARENT
join report_lineage RL on PARENT.report_id = RL.parent_report_id
join report_file CHILD on CHILD.report_id = RL.child_report_id
where PARENT.report_id in
(
  -- this nested query is just the list of *all* reports supposed to go out in last week                                                                      
  select report_id
  from report_file RF
  join action A on A.action_id = RF.action_id
  where
    (A.action_name = 'batch' or RF.next_action = 'send')
    and RF.created_at > now() - interval '2 days'
)
order by CHILD.created_at desc
```